docs/create/mounts.md

```diff
@@ -0,0 +1,8 @@
+---
```

Comment:
These need to be filled in. There are new NFT types coming too, such as shaders.